screens/Meal.tsx

```diff
@@ -1,210 +1,303 @@
+import AsyncStorage from '@react-native-async-storage/async-storage';
 import {BottomTabScreenProps} from '@react-navigation/bottom-tabs';
-import React, {useState} from 'react';
+import React, {useState, useEffect} from 'react';
 import {StyleSheet} from 'react-native';
 import {RootTabList} from '../App';
-import {Box, Center, HStack, ScrollView, Text, VStack} from 'native-base';
-import _ from 'lodash';
+import {
+  Box,
+  Center,
+  HStack,
+  ScrollView,
+  Text,
+  VStack,
+  Modal,
+  Button,
+} from 'native-base';
+import _, {size} from 'lodash';
+import axios from 'axios';
+import {parse} from 'node-html-parser';
 
 type Props = BottomTabScreenProps<RootTabList, 'Meal'>;
 
 export default function Meal({navigation}: Props) {
-  // todo
-  //   const favoritedNumbers = DUMMY_MEAL_INFO.map(({isFavorited}) =>
-  //     Number(isFavorited),
-  //   ).reduce((sum, currValue) => sum + currValue);
-  //   const notFavoritedNumbers = DUMMY_MEAL_INFO.length - favoritedNumbers;
+  type TodaysMenu = {
+    [name: string]: {
+      breakfast: string;
+      lunch: string;
+      dinner: string;
+      contact: string;
+    };
+  };
+  // state 선언
+  const [menu, setMenu] = useState<TodaysMenu | null>(null); // store menu data here
+  const [isFavorite, setIsFavorite] = useState<{} | null>(null); // store favorite or not here
```

Comment:
```suggestion
  const [isFavorite, setIsFavorite] = useState<Record<string, boolean>>(); // store favorite or not here
```
[minor] 타입을 {}로 하는 것은 오브젝트인 모든 타입을 다 허용하는 것이라 실수할 여지가 많아서 위와 같이 해주면 좋을듯

Reply:
ㅇㅎ 저렇게 하면 되는군 ㄱㅅ